clowdapp.yaml

```diff
@@ -101,6 +101,22 @@ objects:
       minReplicas: ${{MIN_REPLICAS}}
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG}
+        livenessProbe:
+          exec:
+            command:
+            - /usr/bin/env
+            - sh
+            - check_cleaner_liveness_probe.sh
+          initialDelaySeconds: 10
+          periodSeconds: 1200
```

Comment:
20min isn't that too much for expired-cleaner-service? Note that there are couple of cycles before the service is considered lifeless. Check out `failureThreshold` on [`core.Probe`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#probe-v1-core), which defaults to 3.

Reply:
20 minutes is default for `expired_cleaner_sleep_minutes` - https://github.com/RedHatInsights/historical-system-profiles-backend/pull/234/files#diff-f8f1173880fcb4a20eaa4751d764b459cf8b8e3e6c9dfe57bd01f50a00cd3962R120 - that means this script just sleeps for 20 minutes and then runs its code.

It was not read from clowder config at all before. I can modify the `periodSeconds` to use the value of `EXPIRED_CLEANER_SLEEP_MINUTES`.